docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
     build:
       dockerfile: RemoteSettings.Dockerfile
       target: local
+      context: .
```

Comment:
For some reason, GHA threw an error without this option set in the Dockerfile (same for the tests container)